pyzeebe/task/task.py

```diff
@@ -1,24 +1,16 @@
-from typing import Callable, List, Dict
+from dataclasses import dataclass
+from typing import Callable
 
-from pyzeebe.decorators.zeebe_decorator_base import ZeebeDecoratorBase
-from pyzeebe.job.job import Job
-from pyzeebe.task.exception_handler import ExceptionHandler
+from pyzeebe.task.task_config import TaskConfig
+from pyzeebe.task.types import JobHandler
 
 
-class Task(ZeebeDecoratorBase):
-    def __init__(self, task_type: str, task_handler: Callable[..., Dict], exception_handler: ExceptionHandler,
-                 timeout: int = 10000, max_jobs_to_activate: int = 32, variables_to_fetch: List[str] = None,
-                 before: List = None, after: List = None):
-        super().__init__(before=before, after=after)
+@dataclass
```

Comment:
I think we need to wait with `dataclasses` for now as it's not supported in 3.6 (EOL in 9 months). 

Or we could introduce https://github.com/ericvsmith/dataclasses as a a dependency – think it's possible in the setup.py to say if 3.6, install dependency.

It simplifies a few things, but we only use it in two places, so we don't save that much on using them.